src/factory.rs

```diff
@@ -543,6 +546,281 @@ impl Factory {
         self.request_texture(Path::new(path_str))
     }
 
+    /// Loads a `glTF` texture.
+    fn load_gltf_texture(
```

Comment:
would be great to move all the `load_gltf` things out in a sub-module
you can still do `impl super::Factory {...}` from there